### PR TITLE
Fix all outstanding ESLint errors

### PR DESCRIPTION
**Pull request by MattIPv4**

### CODE REVIEW

**Review by jacobmitchell04 — APPROVED** [review]
All looks good to me.

File: src/Routes/Lists.js [hunk 1]
```diff
@@ -402,8 +403,8 @@ class ListsRoute extends BaseRoute {
                 });
                 try {
                     await require('../Util/updateIcon')(this.client, this.db, lists[0]);
-                    res.render('error', { title: 'Success', status: 200, message: 'Icon has been updated' })
-                } catch {
+                    res.render('error', { title: 'Success', status: 200, message: 'Icon has been updated' });
+                } catch (e) {
```
Comment (MattIPv4) [inline src/Routes/Lists.js]: Set ecmaVersion to 10 and this change won't be needed.

File: src/Structure/Cache.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ class Cache {
     async get(path) {
         try {
             return (await this.db.select().from('cache').where({ route: path }))[0];
-        } catch {
+        } catch (e) {
```
Comment (MattIPv4) [inline src/Structure/Cache.js]: Set ecmaVersion to 10 and this change won't be needed.

File: src/Structure/Cache.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ class Cache {
     async deleteExpired() {
         try {
             return await this.db('cache').where('expiry', '<', new Date().getSeconds()).del();
-        } catch {
+        } catch (e) {
```
Comment (MattIPv4) [inline src/Structure/Cache.js]: Set ecmaVersion to 10 and this change won't be needed.

File: src/Util/getListFeature.js [hunk 4]
```diff
@@ -6,4 +6,4 @@ module.exports = async (db, id) => {
     } catch (e) {
```
Comment (MattIPv4) [inline src/Util/getListFeature.js]: Set ecmaVersion to 10 and this can not have `(e)`.

File: src/Util/updateListMessage.js [hunk 5]
```diff
@@ -16,4 +16,4 @@ module.exports = async (client, db, list, newListID) => {
     } catch (e) {
```
Comment (MattIPv4) [inline src/Util/updateListMessage.js]: Set ecmaVersion to 10 and this can not use `(e)`.

File: src/Website.js [hunk 6]
```diff
@@ -66,14 +70,18 @@ class Website {
...
                         if (i + 1 === routes.length) {
-                            this.app.use((err, req, res, next) => {
+                            // eslint-disable-next-line no-unused-vars
+                            this.app.use((err, req, res, _) => {
                                 if (req.method === 'POST') {
                                     try {
                                         JSON.parse(req.body);
-                                    } catch(_) {
-                                        return res.status(400).json({ error: true, status: 400, message: 'Body is not parsable JSON' })
+                                    } catch (_) {
```
Comment (MattIPv4) [inline src/Website.js]: Set ecmaVersion to 10 and this can not use `(_)`.

**Review by MattIPv4 — COMMENTED** [review]
catch optional binding changes

**Pull request merged**